ql/experimental/callablebonds/discretizedcallablefixedratebond.cpp

```diff
@@ -73,10 +73,23 @@ namespace QuantLib {
                     couponAdjustments_[j] = CouponAdjustment::pre;
 
                     /* We snapped the callabilityTime so we need to take into account the missing
-                     * discount factor. */
-                    auto discountTillCallDate = termStructure->discount(callabilityDate);
-                    auto discountTillCouponDate = termStructure->discount(couponDate);
-                    adjustedCallabilityPrices_[i] *= discountTillCallDate / discountTillCouponDate;
+                     * discount factor including any possible spread e.g. set in the OAS
+                     * calculation. */
+                    auto spread  = arguments_.spread;
+                    auto calcDiscountFactorInclSpread = [&termStructure, spread](Date date) {
+                        auto time = termStructure->dayCounter().yearFraction(
+                            Settings::instance().evaluationDate(), date);
+                        auto zeroRateInclSpread =
+                            termStructure->zeroRate(date, termStructure->dayCounter(), Continuous,
+                                                    Annual) +
```

Comment:
`Annual` works but it can be confusing to read together with `Continuous`.  Maybe `NoFrequency`?